File: solutions/1944-truncate-sentence/solution.py
class Solution(object):
    def truncateSentence(self, s, k):
        """
        :type s: str
        :type k: int
        :rtype: str
        
        p=""
        count=0
        for i in range(0,len(s)):
            if s[i]==" ":
                count+=1
            if count==k:
                return p
            else:
                p+=s[i]
        return p
        """
        ls=s.split()
        return " ".join(ls[:k]) 

File: solutions/1944-truncate-sentence/solution.java
class Solution {
    public String truncateSentence(String s, int k) {
        List<String>ls=new ArrayList<>();
        String[] a=s.split("\\ ");
        for(int i=0;i<a.length;i++)
        {
            if(i!=k)
                ls.add(a[i]);
            else
                break;
        }
        return String.join(" ",ls);
    }
}
